src/cpp/session/resources/schema/user-prefs-schema.json

```diff
@@ -617,7 +617,7 @@
         },
         "shiny_viewer_type": {
             "type": "string",
-            "enum": ["user", "none", "pane", "window", "browser"],
+            "enum": ["user", "none", "pane", "tutorial", "window", "browser"],
```

Comment:
I don't think this should be part of the pref enum because it's nonsensical for people to prefer to have their Shiny apps open in the Tutorial pane.